module/completeness/src/Resources/config/services.yml

```diff
@@ -22,8 +22,8 @@ services:
     resource: '../../Application/Controller/*'
     tags: [ 'controller.service_arguments' ]
 
-  Ergonode\Completeness\Domain\Provider\TemplateElementCompletenessStrategyProvider:
-    - '@Ergonode\Completeness\Domain\Calculator\Strategy\AttributeTemplateElementCompletenessStrategy'
-    - '@Ergonode\Completeness\Domain\Calculator\Strategy\UiTemplateElementCompletenessStrategy'
+  Ergonode\Completeness\Infrastructure\Persistence\Projector\:
+    resource: '../../Infrastructure/Persistence/Projector/*'
+    tags: [ 'ergonode.event_sourcing.projector' ]
```

Comment:
Wouldn't it be good to use instaceof if the this is just a local tag? Potentially there could additional nested services.